docs/docs/.vuepress/config.js

```diff
@@ -18,195 +19,8 @@ module.exports = {
       },
     ],
   ],
-  themeConfig: {
-    algolia: {
-      appId: 'XY28M447C5',
-      apiKey: "9d9169458128b3d60c22bb04da4431c7",
-      indexName: "pipedream",
-      algoliaOptions: {
-        facetFilters: ['version:latest']
-      }
-    },
-    searchPlaceholder: "Search...",
-    logo: "/pipedream.svg",
-    repo: "PipedreamHQ/pipedream",
-    nav: [
-      {
-        text: "v2",
-        className: "docs-version",
-        ariaLabel: "Docs Version Menu",
-        items: [
-          {
-            text: "v2",
-            link: "https://pipedream.com/docs",
-            internal: true,
-            badge: "New",
-            badgeVariation: "primary",
-          },
-          {
-            text: "v1",
-            internal: true,
-            link: "https://pipedream.com/docs/v1",
-          },
-        ],
-      },
-    ],
-    smoothScroll: false,
-    // if your docs are not at the root of the repo:
-    docsDir: "docs/docs",
-    editLinks: true,
-    // custom text for edit link. Defaults to "Edit this page"
-    editLinkText: "Help us improve this page! Submit an edit on Github",
-    sidebar: [
-      "/",
-      "/quickstart/",
-      {
-        title: "Workflows",
-        children: 
-        [
-          "/workflows/",
-          "/workflows/steps/",
-          "/workflows/steps/triggers/",
-          "/workflows/events/",
-          "/components/actions/",
-          "/workflows/concurrency-and-throttling/",
-          "/environment-variables/",
-          "/workflows/settings/",
-          "/workflows/networking/",
-          "/migrate-from-v1/",
-        ]
-      },
-      "/sources/",
-      "/connected-accounts/",
-      "/user-settings/",
-      {
-        title: "Reference: Code, APIs + CLI",
-        children: [
-          {
-            title: "Writing Code in Workflows",
-            type: "group",
-            initialOpenGroupIndex: 1,
-            children: [
-              "/code/",
-              {
-                title: "Node.js",
-                type: "group",
-                children: [
-                  "/code/nodejs/",
-                  "/code/nodejs/auth/",
-                  "/code/nodejs/http-requests/",
-                  "/code/nodejs/working-with-files/",
-                  "/code/nodejs/async/",
-                ],
-              },
-              "/code/python/",
-              "/code/go/",
-              "/code/bash/",
-              "/destinations/",
-              "/scheduling-future-tasks/",
```

Comment:
This was removed, but I frequently point users here, so I think it should be discoverable in the nav